qemu/tests/block_copy.py

```
@@ -298,7 +297,10 @@ def action_when_start(self):
                 bg = utils.InterruptedThread(fun)
                 bg.start()
                 if bg.isAlive():
-                    self.processes.append(bg)
+                    try:
```

For the try, just want to ignore all exceptions.